lib/tasks/gettext.rake

```diff
@@ -52,6 +52,68 @@ namespace :gettext do
            files_to_translate.join(' '))
   end
 
+  def metadata_pot_file_path
```

Comment:
Rather, move it into an `.rb` file for ease of testing. The other PR in this repo creates a `task_helper.rb` file, which seems a sensible way about it.